### PR TITLE
Completed css-two-column-layout

### PR DESCRIPTION
**Pull request by jdfenlason**
Screen Shot 2021-03-14 at 11 20 37 AM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  align-content: center;
+  align-items: center;
+}
+.justify-center {
+max-width: 189px;
+}
+
+button {
+  padding: 5px 20px;
+  margin-bottom 10px;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:98]: You are missing a colon on this line

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ max-width: 150px;
+}
+.wrapper > * {
+justify-content: center;
+  align-content: center;
+  align-items: center;
+}
+.justify-center {
+max-width: 189px;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:94]: Make sure your indenting is consistent.

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  margin-bottom: 0px;
+}
+p {
+  justify-content: flex-start;
+  font-size: 10px;
+line-height: 1.5em;
+max-width: 250px;
+color: rgb(79, 79, 79)
+
+
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:63]: indenting needs adjustment

File: css-two-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+.container{
+
+width: 700px;
+margin: auto;
+background-color: white;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:6]: indenting needs adjustment

File: css-two-column-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+}
+.row {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+
+
+
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:18]: Please remove empty lines in your CSS rules, with the exception of one empty line between different rules.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! Please see my notes for detailed feedback. Additionally, your screenshot looks like the two column section is not centered in the same way as the header, please try to make sure these are consistent, and give me an updated screenshot when you have updated it.

**Comment by jdfenlason** [comment]
Screen Shot 2021-03-15 at 10 51 12 PM

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**